cranker/src/lib.rs

```diff
@@ -0,0 +1,392 @@
+use ::{
+    solana_sdk::{
+        pubkey::Pubkey,
+        signature::Keypair,
+        signer::Signer,
+        transaction::Transaction,
+        compute_budget::ComputeBudgetInstruction,
+        commitment_config::{ CommitmentConfig, CommitmentLevel },
+    },
+    solana_client::{
+        rpc_client::RpcClient,
```

Comment:
Let's update to use the nonblocking client, similar to how Jito's restaking cranker is setup https://github.com/jito-foundation/restaking/blob/master/crankers/src/core.rs#L3